src/components/Boxs/BoxBase/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box as MaterialBox, BoxProps as MaterialBoxProps } from "@material-ui/core";
+
+export interface BoxBaseProps extends MaterialBoxProps {
+    display?: "flex" | "inline" | "block" | "inline-block";
+    flexDirection?: "row" | "row-reverse" | "column" | "column-reverse" | "initial" | "inherit";
+    flexWrap?: "nowrap" | "wrap" | "wrap-reverse" | "initial" | "inherit";
+    justifyContent?:
+        | "flex-start"
+        | "flex-end"
+        | "center"
+        | "space-between"
+        | "space-around"
+        | "space-evenly"
+        | "inital"
+        | "inherit";
+    alignItems?:
+        | "stretch"
+        | "center"
+        | "flex-start"
+        | "flex-end"
+        | "baseline"
+        | "initial"
+        | "inherit";
+    bgcolor?: "primary" | "secondary" | "info" | "error" | "warning" | "success";
+    colorProps?: "main" | "light" | "dark";
+    boxSizing?: "content-box" | "border-box";
+    m?: number;
+    p?: number;
+    mt?: number;
+    mr?: number;
+    mb?: number;
+    ml?: number;
+    pt?: number;
+    pr?: number;
+    pb?: number;
+    pl?: number;
+    width?: number;
+    height?: number;
+    minWidth?: number;
+    maxWidth?: number;
+    minHeight?: number;
+    maxHeight?: number;
+}
```

Comment:
I like your effort but have you checked the material interfaces for Box? 
Please explore the typings of Box by **CTRL click** on it and use the pre-written types appropriately
node_modules\@material-ui\system\index.d.ts
![image](https://user-images.githubusercontent.com/26225039/117536907-f5dc4a00-b027-11eb-8a05-bbf705f43bf2.png)